superdesk/publish/formatters/ninjs_formatter.py

```diff
@@ -142,7 +142,7 @@ def __init__(self):
         self.format_type = "ninjs"
         self.can_preview = True
         self.can_export = True
-        self.internal_renditions = ["original"]
+        self.internal_renditions = ["original", "baseImage"]
```

Comment:
if we send those 2 we can as well send them all based on the configured renditions in settings, but would be probably good to introduce a new config for this so we can revert to the current behaviour if needed:
```py
NINJS_COMMON_RENDITIONS = list(RENDITIONS.keys())
```